pkgs/tools/system/kanata/default.nix

```diff
@@ -8,19 +8,19 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "kanata";
-  version = "1.6.1";
+  version = "1.7.0-prerelease-1";
 
   src = fetchFromGitHub {
     owner = "jtroo";
     repo = pname;
```

Comment:
```suggestion
    repo = "kanata";
```

Reply:
Thanks, I will change it in a following PR.

Reply:
Hmm, actually I do not think use `rec` and `repo = pname` is an anti-pattern.